app/src/main/java/com/jerboa/ui/theme/Theme.kt

```diff
@@ -34,7 +40,20 @@ fun JerboaTheme(
 
     // Dynamic schemes crash on lower than android 12
     val dynamicPair = if (android12OrLater) {
-        Pair(dynamicLightColorScheme(ctx), dynamicDarkColorScheme(ctx))
+        val jerboaImageHighlight = Color(0xCCD1D1D1)
```

Comment:
This is wild that add custom colors requires this much code bloat. Is it really that necessary? Now all theme designers have to hack on these colors too, rather than just using the defaults.

The tradeoff of this massive code bloat is not worth it IMO. If our three options are:

- Use the existing colors: primary, tertiary, etc, but made semi-transparent for these carets.
- Less optimal - hard code the colors
- Add these custom colors into every theme, and extend jetpack-compose theming like you've done here.

Please use option 1 or 2, this extra complication is not worth it at all.

Reply:
Option 1 definitely won't work; we need new colors here (see https://github.com/dessalines/jerboa/pull/932#discussion_r1251311355 for my reasoning why).

A note on the implementation: Designers will still be able to use the default colors through `MaterialTheme.colorScheme`; it's just the new `MaterialTheme.jerboaColorScheme` that provides custom colors like these. I don't believe this issue will be unique; there may very well be other times we need colors that aren't present in Material Design 3. We can definitely hardcode each new color, but they will never be compatible with the theming system that users can select, if we do that.

I do agree that it's kind of bloaty, though. I'm fine with reverting to option 2 if it comes down to that.

Thoughts from @MV-GH since they requested the color theming?

Reply:
I was fine with the hard coding tbh. But since we already have this implemented with the theme might as well keep it. @twizmwazin wanna be the tie braker?